View/ViewHandler.php

```diff
@@ -448,6 +449,18 @@ private function initResponse(View $view, $format)
             if ($serializer instanceof SerializerInterface) {
                 $context = $this->getSerializationContext($view);
                 $content = $serializer->serialize($data, $format, $context);
+            //       test for symfony 2.7 version 
+            } elseif (class_exists('\Symfony\Component\Serializer\Normalizer\ObjectNormalizer') && $serializer instanceof SymfonySerializerInterface) {
+                    
+                $context = $this->getSerializationContext($view);
```

Comment:
puh .. this is getting a bit nasty. we then also need such a method on the ``View`` along with a ``setSymfonySerializationContext()`` method. very ugly but I see no way to not have to duplicate those methods without breaking BC in a big way.